src/js/utils/fn.js

```diff
@@ -60,19 +60,15 @@ export const bind_ = function(context, fn, uid) {
  *
  * @return   {Function}
  */
-export const throttle = function(fn, wait) {
-  let last = window.performance.now();
```

Comment:
This was changed to use `window.performance.now()` in #5870. Why do you propose changing it to use `Date.now()`?

Reply:
That's a good question, we use `window.performance.now()` when we need a high level of accuracy as it is a measurement of floating point milliseconds, but do these `0.153184` milliseconds really matter in our situation? I don't think they matters because we are using 30 milliseconds as a delay for the throttle which is so huge compared to `% 1`, so we can simply neglect these neglectable numbers which have bad small effect on the performance, as `window.performance.now()` is `3 to 4` times slower than `Date.now()`, as shown in the following picture:
![Screenshot 2023-08-11 162527](https://github.com/videojs/video.js/assets/64714761/a89c5f0a-3973-4d2d-a319-21c1268e55b7)
Finally, I have initialized `lastCallTime` with 0, to avoid calling any function that requires a non-zero milliseconds to run.

Reply:
The main reason to use performance.now() here isn't the higher resolution but the guarantee of monotonic time updates. I would be inclined to leave it as is due to that.

I'm not sure that the performance difference between Date.now() and performance.now() in this usage is applicable.

Reply:
Perhaps the method used to determine the start time should be configurable, I can imagine `performance.now()` being superceded by some other better native JS alternative at some point (it never stops, does it) so a method that allows the user to use `Date.now()` or `performance.now()` or even `nextnewthing.now()`, is a reasonable idea, one I should've thought of when I submitted #5870